.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
This file is not meant for this pr right? it looks like a file that was pushed before you added .vscode/ to the .gitignore

Reply:
Good catch, going to change that